addons/account/models/account_journal.py

```diff
@@ -57,33 +57,39 @@ def _default_alias_domain(self):
     account_control_ids = fields.Many2many('account.account', 'journal_account_control_rel', 'journal_id', 'account_id', string='Accounts Allowed',
         check_company=True,
         domain="[('deprecated', '=', False), ('company_id', '=', company_id)]")
-    default_credit_account_id = fields.Many2one('account.account', string='Default Credit Account', copy=False,
-        domain=[('deprecated', '=', False)], help="It acts as a default account for credit amount",
-        check_company=True,
-        ondelete='restrict')
-    default_debit_account_id = fields.Many2one('account.account', string='Default Debit Account', copy=False,
-        check_company=True,
-        domain="[('deprecated', '=', False), ('company_id', '=', company_id)]", help="It acts as a default account for debit amount", ondelete='restrict')
-    payment_debit_account_id = fields.Many2one('account.account', string='Outstanding Receipts Account', copy=False,
-        ondelete='restrict',
-        domain=[('deprecated', '=', False)],
+    default_account_type = fields.Many2one('account.account.type', compute="_compute_default_account_type", default=True)
```

Comment:
default=True for a computed many2one ?

Reply:
Hi @nseinlet, Thanks for pointing this out. It is the remnant of a mistake I made when creating this field. I have removed the default since the field is computed anyway.